src/templates/Navigator/navigator.module.scss

```diff
@@ -0,0 +1,32 @@
+@import 'src/assets/stylesheets/global';
+
+.navigator {
+  // Place your component specific scss here.
+  background-color: $neutral-7;
+  height: 100%;
+  display: flex;
+}
+
+.left-rail {
+  align-self: flex-stretch;
+  width: ru(11);
```

Comment:
Not really.. if they don't give us a grid with the layout, I don't think we should bother applying it.